# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/xul/test/browser_bug706743.js
# Commit: beeda5ef3712
# Full Hash: beeda5ef3712b73a19c7a4ce08a52ab4dd08b802
# Author: David P <daparks@mozilla.com>
# Date: 2024-07-04 16:09:52
# Regressor Bug: 1893119
# File Overlap Count: 1
# Description:
#   Bug 1893119: Part 20 - Move EndDragSession from nsIDragService to nsIDragSession r=gstoll,rkraesig,win-reviewers,geckoview-reviewers,m_kato
#   
#   This is a straightforward move.  It does take the liberty of breaking out an
#   EndDragSessionImpl method, which will be needed later.
#   
# ==============================================================================

diff -r 6c3ffc8c534c -r beeda5ef3712 layout/xul/test/browser_bug706743.js
--- a/layout/xul/test/browser_bug706743.js	Thu Jul 04 07:48:10 2024 +0000
+++ b/layout/xul/test/browser_bug706743.js	Thu Jul 04 07:48:11 2024 +0000
@@ -79,7 +79,7 @@
     await new Promise(resolve => setTimeout(resolve, 100));
   } finally {
     removeEventListener("popupshown", tooltipNotExpected, true);
-    dragService.endDragSession(true);
+    dragService.getCurrentSession().endDragSession(true);
   }
 
   await BrowserTestUtils.synthesizeMouse(